### PR TITLE
Dependency updates: week of 11/07/22

### PR DESCRIPTION
**Pull request by TomElliottFlexion**
### Notes:
- Updated Terraform to 1.3.4
- Published a new ECR image 2.11.0 (this will require manually running `docker-to-ecr.sh` on the court side)
- [Migration](https://app.circleci.com/pipelines/github/flexion/ef-cms/39220/workflows/b06203d5-c6b8-423f-90ec-cea88170f10c)
  
### Minor/patch package update(s): 
 - @aws-sdk/credential-provider-node
 - aws-sdk
 - @typescript-eslint/eslint-plugin
 - @typescript-eslint/parser 
 - @wojtekmaj/enzyme-adapter-react-17
 - artillery
 - babel-jest
 - csv-parse
 - eslint
 - eslint-plugin-jest
 - husky
 - jest
 - jest-environment-jsdom
 - jest-environment-node
 - sass
 - sass-loader
 - webpack
 
### Security audit fixes, secure versions added to resolutions block:
  -   loader-utils to 2.0.3

### CODE REVIEW

**Review by jasonfu9966 — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ commands:
 jobs:
   build-client-integration:
     docker:
-      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.9.0
+      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.11.0
```
Comment (rachelschneiderman) [inline .circleci/config.yml:66]: Was this the only job whose container version was updated on purpose?

File: docs/additional-resources/testing.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ At the end of a sprint PR, we'd like to run manual smoke tests over our system i
 - sign the pdf
 - add a docket entry
 - serve
-- verify the petitioner got an email notificaton
+- verify the petitioner got an email notification
```
Comment (rachelschneiderman) [inline docs/additional-resources/testing.md:77]: Nice catch

File: docs/terraform.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ These various terraform deployment will be explained in detail later in the foll
 │   ├── variables.tf // a definition of all variables of this terraform directory
 ```
 
-The `variables.tf` file is used to defined the arguments needed to run terraform in order to successfully deploy the application.  These variables are then accessibility to other sibiling `.tf` files and can be referenced via `var.my_variable_name`, or interpolated in strings using `"testing-${var.my_variable_name}"`.  
+The `variables.tf` file is used to defined the arguments needed to run terraform in order to successfully deploy the application.  These variables are then accessibility to other sib ling `.tf` files and can be referenced via `var.my_variable_name`, or interpolated in strings using `"testing-${var.my_variable_name}"`.
```
Comment (rachelschneiderman) [inline docs/terraform.md]: Accidental space in 'sibling'?

File: .circleci/config.yml (hunk 1 above)
Comment (TomElliottFlexion) [reply .circleci/config.yml:66]: Holy cow, nope! Good catch, thank you!

Comment (TomElliottFlexion) [reply .circleci/config.yml:66]: Deploying to exp5 again

File: docs/terraform.md (hunk 3 above)
Comment (TomElliottFlexion) [reply docs/terraform.md]: Indeed there is, thanks!

File: .circleci/config.yml [hunk 4]
```diff
@@ -129,7 +129,7 @@ jobs:
 
   build-ts-api:
     docker:
-      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.4.0
+      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.11.0
```
Comment (TomElliottFlexion) [inline .circleci/config.yml:132]: Going to give it a try

File: .circleci/config.yml (hunk 1 above)
Comment (TomElliottFlexion) [reply .circleci/config.yml:66]: https://app.circleci.com/pipelines/github/flexion/ef-cms/39253/workflows/c6cc905d-1ed4-413e-9212-3dee6178cb7f

**Pull request merged**